coriolis/utils.py

```diff
@@ -336,6 +336,9 @@ def check_fs(ssh, fs_type, dev_path):
         LOG.debug("File system checked:\n%s", out)
     except Exception as ex:
         LOG.warn("Checking file system returned an error:\n%s", str(ex))
+        LOG.warn(
```

Comment:
There is no real reason for this message to be emitted here so please remove it.

Please add try-excepts and wrapper messages to wherever this is called within the `coriolis.osmorphing.osmount` module.